robottelo/datafactory.py

```diff
@@ -181,6 +195,32 @@ def valid_environments_list():
 
 
 @datacheck
+def valid_user_data_list(size=10):
```

Comment:
In my opinion, we don't need that method at all and use `valid_usernames_list` or `generate_strings_list` or any other suitable methods that we already have.
Then `test_positive_create` test should be removed and all things mentioned there should be tested separately (as you already did).
Testing all combination is excessive in that case as add unnecessary load and time on execution procedure, adding almost none benefits